packages/mco/models/argo-cd.ts

```diff
@@ -0,0 +1,13 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
+export const ArgoApplicationSetModel: K8sModel = {
+  label: 'ApplicationSet',
+  labelPlural: 'ApplicationSets',
+  apiVersion: 'v1alpha1',
+  apiGroup: 'argoproj.io',
+  plural: 'applicationsets',
+  abbr: 'APPSET',
```

Comment:
nits:
how about this only ??
```suggestion
  abbr: 'AS',
```